### DISCUSSION TITLE
Postgresql plugin with custom port

### DISCUSSION THREAD

**Discussion by vineeth-brt** [ArcadeData/arcadedb#657] [Q&A] [answered]
Hi 

I have started Arcadedb with postgresql plugin enabled. Since a postgresql instance is already running in my machine,  it shows an error ' Unable to listen for connections using the configured ports '5432' on host '0.0.0.0' . Is there any option to enable postgresql plugin in a custom port other than 5432 on arcadedb startup.

Regards
Vineeth

**Comment by lvca** [comment, marked as answer]
You can configure the port to be used with `-Darcadedb.postgres.port=3333`. You can also change the default `0.0.0.0` by setting `arcadedb.postgres.host`

**Reply by vineeth-brt** [reply]
Thank you @lvca
